homeassistant/components/media_player/gpmdp.py

```diff
@@ -200,22 +201,31 @@ def send_gpmdp_msg(self, namespace, method, with_id=True):
     def update(self):
         """Get the latest details from the player."""
         time.sleep(1)
```

Comment:
What??? This should be removed.

Reply:
I'm not sure what's that for, but I just assumed the person who first made the component had a reason? If you think it's useless then I can remove it.

Reply:
This was introduced with https://github.com/home-assistant/home-assistant/pull/3115